Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -3791,5 +3844,15 @@ public int SelectedRowIndex
     /// </summary>
     [Parameter] public bool AutoGenerateColumns { get; set; } = true;
 
+    /// <summary>
+    /// Gets or sets DataGridEditMode options.
+    /// </summary>
+    [Parameter] public DataGridEditModeOptions DataGridEditModeOptions { get; set; }
+
+    /// <summary>
+    /// Gets or sets whether the Datagrid is Cell Navigable, users will be able to navigate the Grid cells by pressing the Keyboard's ArrowLeft, ArrowUp, ArrowRight and ArrowDown keys.
+    /// </summary>
+    [Parameter] public bool CellNavigable { get; set; }

```

Comment:
We now have two APIs, CellNavigable and Navigable. Both are bool. I think we need to make them more coherent.

I propose to have `DataGridNavigationMode NavigationMode` with

- `Cell`
- `Row`

The reason is that having both might be confusing and are too similar by name. What happens if user have bth enabled?

So we need to obsolete `Navigable`, and introduce this new API.

Reply:
For the row you mean up, down? :)
Your suggestion seems like a nice idea.

Reply:
Yes, Row would behave the same as existing Navigable. Pressing Up/Down will focus rows.